adobe_vipm/flows/utils.py

```diff
@@ -17,10 +17,10 @@
 from adobe_vipm.utils import find_first
 
 
-def get_parameter(object, parameter_phase, parameter_name):
+def get_parameter(order, parameter_phase, param_external_id):
```

Comment:
we can left here object instead of order since the function applies both to orders and subscriptions, right ?

Reply:
Agree :-)